ods_ci/tests/Tests/1300__model_registry/1302_model_registry_model_serving.robot

```diff
@@ -248,3 +250,25 @@ Run Update Notebook Script
     Log    ${result.stdout}
     Log    ${result.stderr}
     Should Contain    ${result.stdout}    Modified notebook saved
+
+Remove Model Registry
+    [Documentation]    Run multiple oc delete commands to remove model registry components
+    Run OC Delete Command    oc delete smm default -n ${NAMESPACE_MODEL-REGISTRY}
+    Run OC Delete Command    oc delete -k ${MODELREGISTRY_BASE_FOLDER}/samples/secure-db/mysql-tls
+    Run OC Delete Command    oc delete secret modelregistry-sample-grpc-credential -n ${NAMESPACE_ISTIO}
+    Run OC Delete Command    oc delete secret modelregistry-sample-rest-credential -n ${NAMESPACE_ISTIO}
+    Run OC Delete Command    oc delete namespace ${NAMESPACE_MODEL-REGISTRY} --force
+
+Run OC Delete Command
```

Comment:
Can you please rename this Keyword to something like `Run And Verify Command` and place it in the `ods_ci/tests/Resources/Common.robot` file, above the similar KW `Run And Watch Command` (line 435) ?

Reply:
![Screenshot 2024-08-15 at 16 21 02](https://github.com/user-attachments/assets/12ef8ec6-ebe8-44df-8648-9f6c166623e0)

Renamed keyword and moved it to Common.robot